server/get_errors.jl

```diff
@@ -0,0 +1,18 @@
+import Base.in
+
+BestResult = @NamedTuple{shifts::Shifts, score::Number}
```

Comment:
Nie wiadmo co to jest `Shifts` w tym pliku, nie zakładałbym, że to będzie zaincludowane wcześniej.